libs/ledger-live-common/src/families/cardano/js-getTransactionStatus.ts

```diff
@@ -220,6 +220,19 @@ const getTransactionStatus = async (
     throw new AccountAwaitingSendPendingOperations();
   }
 
+  if (a.cardanoResources.utxos.length === 0) {
```

Comment:
So this error happened when user had no funds & no history of txs on the account ?

Reply:
no funds yes but tx history can be there. so it happens when user has no utxos and has rewards

Reply:
Ok, thanks ! Can you add a unit test for this case ?